docs/building-deployer-helm.md

```diff
@@ -623,69 +610,71 @@ general
 [onboarding guide](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/set-up-environment#product-identifiers).
 See [this section](building-deployer.md#publishing-a-new-version)
 for more information about tags. Let's say your application has a
-track named `v6`.
+track named `v6.0.0`.
 
 Your GCR repo should have the following images:
 
 ```text
-gcr.io/your-company/wordpress:v6
-gcr.io/your-company/wordpress/deployer:v6
-gcr.io/your-company/wordpress/mariadb:v6
+gcr.io/your-company/wordpress:v6.0.0
+# The deployer excludes the patch version in SemVer
+gcr.io/your-company/wordpress/deployer:v6.0
+gcr.io/your-company/wordpress/mariadb:v6.0.0
 ```
 
 Then your schema should look similar to the following:
 
 ```yaml
-properties:
-  image.primary:
-    type: string
-    default: gcr.io/your-company/wordpress:v6
-    x-google-marketplace:
-      type: IMAGE
-  image.database:
-    type: string
-    default: gcr.io/your-company/wordpress/mariadb:v6
-    x-google-marketplace:
-      type: IMAGE
-```
-
-The default values in your schema are important, as this
-is how Marketplace knows where to find the application images.
+  images:
+    '':
+      properties:
+        wordpress.image.registry:
+          type: REGISTRY
+        wordpress.image.repository:
+          type: REPO_WITHOUT_REGISTRY
+        wordpress.image.tag:
+          type: TAG
+    mariadb:
+      properties:
+        wordpress.mariadb.image.registry:
+          type: REGISTRY
+        wordpress.mariadb.image.repository:
+          type: REPO_WITHOUT_REGISTRY
+        wordpress.mariadb.image.tag:
+          type: TAG
+```
+
+The properties of each image in `images` are used by Marketplace to
+parameterize the images in `values.yaml` for your chart. The name of
+the primary image in `schema.yaml` is typically `''` such that 
```

Comment:
I think, to keep things not confusing, we can remove this empty string usage. It's still supported, but I think it's better to not use it. Such was required in the past due to "primary image" requirement.

Reply:
Fixed